glide.lock

```diff
@@ -51,7 +51,7 @@ imports:
   subpackages:
   - netlabel
 - name: github.com/infobloxopen/infoblox-go-client
-  version: e3afa79f174f4ef5932d24c97d69fdf6569bd3c5
+  version: v0.3.0
```

Comment:
you should not change this file actually? I have a  doubt now Did you tested this changes?

Reply:
I only compiled and built all the go files I did not get any error..